provider/pkg/provider/helm_release.go

```diff
@@ -1441,6 +1455,54 @@ func mapToInterface(in any) any {
 	return in
 }
 
+// searchProgramDirectory implements a best-effort search for a chart in the program directory.
+func searchProgramDirectory(name, version string) (*string, *helmchart.Metadata, bool) {
+	file := fmt.Sprintf("%s.tgz", name)
+	if c, err := loader.LoadFile(file); err == nil {
+		return &file, c.Metadata, true
+	}
+	file = fmt.Sprintf("%s-%s.tgz", name, version)
```

Comment:
Should we prefer returning the `name-version.tgz` chart first?

Reply:
That's reasonable.